packages/material-ui-unstyled/src/SwitchUnstyled/switchUnstyledClasses.ts

```diff
@@ -0,0 +1,30 @@
+import generateUtilityClass from '../generateUtilityClass';
+import generateUtilityClasses from '../generateUtilityClasses';
+
+export interface SwitchUnstyledClasses {
+  root: string;
+  checked: string;
+  disabled: string;
+  input: string;
+  thumb: string;
+  button: string;
+  focusVisible: string;
+}
+
+export type SwitchUnstyledClassKey = keyof SwitchUnstyledClasses;
+
+export function getSwitchUnstyledUtilityClass(slot: string): string {
+  return generateUtilityClass('SwitchUnstyled', slot);
+}
+
+const switchUnstyledClasses: SwitchUnstyledClasses = generateUtilityClasses('SwitchUnstyled', [
```

Comment:
Are we sure about this Unstyled suffix? In practice, the class names will be used to style, so I don't really see the use case. The same classes will also be used by the default styled components.

```suggestion
const switchUnstyledClasses: SwitchUnstyledClasses = generateUtilityClasses('MuiSwitch', [
```